src/pages/DisbursementDraftDetails.tsx

```diff
@@ -176,6 +181,14 @@ export const DisbursementDraftDetails = () => {
     resetState();
   };
 
+  const handleCalculateFutureBalance = (): number => {
+    const assetBalance = Number(
+      allBalances?.find((a) => a.assetCode === draftDetails?.details.asset.code)
+        ?.balance,
+    );
+    return assetBalance - Number(draftDetails?.details.stats?.totalAmount);
```

Comment:
To make sure we don't get NaN if `totalAmount` is undefined.

```suggestion
    return assetBalance - Number(draftDetails?.details.stats?.totalAmount || 0);
```